stdlib/src/socket.rs

```diff
@@ -35,18 +35,13 @@ mod _socket {
     use libc as c;
     #[cfg(windows)]
     mod c {
-        pub use winapi::shared::netioapi::{if_indextoname, if_nametoindex};
-        pub use winapi::shared::ws2def::{
-            INADDR_ANY, INADDR_BROADCAST, INADDR_LOOPBACK, INADDR_NONE,
-        };
-        pub use winapi::um::winsock2::{
-            SO_EXCLUSIVEADDRUSE, getprotobyname, getservbyname, getservbyport, getsockopt,
-            setsockopt,
-        };
-        pub use winapi::um::ws2tcpip::{
-            EAI_AGAIN, EAI_BADFLAGS, EAI_FAIL, EAI_FAMILY, EAI_MEMORY, EAI_NODATA, EAI_NONAME,
-            EAI_SERVICE, EAI_SOCKTYPE,
-        };
+        pub use windows_sys::Win32::NetworkManagement::IpHelper::{if_indextoname, if_nametoindex};
+
+        pub const INADDR_ANY: u32 = 0x00000000;
```

Comment:
This is defined under Win32::Networking::WinSock

https://docs.rs/windows-sys/latest/windows_sys/Win32/Networking/WinSock/constant.INADDR_ANY.html

Reply:
Had to upgrade to windows-sys 0.59 to do it